src/main/java/io/openliberty/tools/common/plugins/util/InstallFeatureUtil.java

```diff
@@ -1170,7 +1168,7 @@ private void installFeaturesOnContainer(List<String> features, boolean acceptLic
         
 
         if(verifyOption != null) {
-            featureUtilityCommand += "--verify=" + verifyOption.name();
+            featureUtilityCommand += " --verify=" + verifyOption.name();
```

Comment:
Can you please fix the line above to add a space before `--acceptLicense`? I know you didn't add that code, but your new option here made me think about concatenating options and there needing to be spaces between them.